spec/validators/vcr_stripe_check_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'current Stripe gem version vs Stripe VCR cassete version', :vcr, :twitter do
```

Comment:
Here is where you added the spec file which you delete in the next commit. Squashing the two commits removes it from the history completely.